test-suite/src/test/java/views/NonHTMLViewRendererTest.java

```diff
@@ -168,7 +167,7 @@ public boolean exists(@NonNull String viewName) {
         }
     }
 
-    @Introspected
+    @Serdeable
```

Comment:
I do not know why I had to use `@Serdeable` in place of `@Introspected`...  Does 4.0.0 of security switch to serde?